src/WORKSPACE

```diff
@@ -171,3 +171,29 @@ http_archive(
     strip_prefix = "buildtools-9e63c6f02e63adaab63e21a3a8feb75f4f7503cb",
     url = "https://github.com/bazelbuild/buildtools/archive/9e63c6f02e63adaab63e21a3a8feb75f4f7503cb.zip",
 )
+
+git_repository(
+    name = "rules_python",
+    commit = "a0fbf98d4e3a232144df4d0d80b577c7a693b570",
+    remote = "https://github.com/bazelbuild/rules_python.git",
+    shallow_since = "1586444447 +0200",
+)
+
+load("@rules_python//python:repositories.bzl", "py_repositories")
+
+py_repositories()
+
+load("@rules_python//python:pip.bzl", "pip_repositories")
+
+pip_repositories()
+
+load("@rules_python//python:pip.bzl", "pip3_import")
+
+pip3_import(
+    name = "python_control_deps",
+    requirements = "//firmware/app/control/python_controller:requirements.txt",
+)
```

Comment:
Is it possible to move both the `load` of `pip3_import` and and the call to it to the `BUILD` files in `python_controller`?

Reply:
Tried copy-pasting all of the pip related commands to the BUILD fail and it didn't compile.

Reply:
confirmed this is the correct usage pattern from the docs: https://github.com/bazelbuild/rules_python/blob/master/docs/pip.md